frontend/src/views/CellGuide/components/LandingPage/index.tsx

```diff
@@ -3,15 +3,46 @@ import CellGuideCardSearchBar from "../CellGuideCardSearchBar";
 import { StyledHeader, Wrapper } from "./style";
 import CellGuideBottomBanner from "../CellGuideBottomBanner";
 import { LANDING_PAGE_HEADER } from "src/views/CellGuide/components/LandingPage/constants";
+import CellGuideMobileHeader from "../CellGuideMobileHeader";
+import { useCallback, useEffect, useMemo, useState } from "react";
+import { throttle } from "lodash";
 
 const TITLE = "CellGuide Cell Types and Cell Tissues - CZ CELLxGENE";
 
 const DESCRIPTION =
   "Explore single-cell transcriptomics data in CellGuide, a comprehensive resource that empowers researchers with deep insights into the intricacies of cell types";
 
 export default function LandingPage(): JSX.Element {
+  // This is for mobile only
+  const [openSearch, setOpenSearch] = useState(false);
```

Comment:
let's add `mobile` to the getter/setter names for clarity, and we can drop the comment as a result too!

E.g.,

`const [openMobileSearch, setOpenMobileSearch] = useState(false);` or something better